specifyweb/frontend/js_src/lib/components/WbPlanView/CustomSelectElement.tsx

```diff
@@ -682,9 +682,7 @@ export function CustomSelectElement({
             {...selectGroupData}
             hasArrow={has('arrow')}
             hasIcon={has('icon')}
-            selectGroupLabel={
-              customSelectSubtype === 'simple' ? selectGroupLabel : undefined
-            }
+            selectGroupLabel={selectGroupLabel}
```

Comment:
looks good, but could you double check that the UI impact of this on the workbench mapper and query builder is ok?

Reply:
yep verified